data/successors/successor ships.txt

```diff
@@ -1025,6 +1032,8 @@ ship "Veusade"
 	gun 11 -64 "Drag Cannon"
 	gun 22 -34 "Drag Cannon"
 	gun 25 -32 "Drag Cannon"
+	pylon -25 -30
+	pylon 25 -30
```

Comment:
And lore (and faction practices in terms of how they would design, build, and utilize ships) is what matters. 

So for this, it's just a question of degrees: Is it a ship that was merely designed ideally to be used with drag cannons; or is it a ship that, as far as they're concerned, might as well have those drag cannons built in because they're never going to be changed?

Reply:
More the latter than the former. The Veusade is "The Ship That Exists to Field Eight Drag Cannons"---it is specifically designed to use exactly one weapon very well. More than the other warships, it is not intended to be flexible nor to have a variety of loadouts. While it _can_ support other weapons because ES is a game where the player has choice over these matters, I do not see the designers as having made non-trivial concessions (as I envision pylons to be) in order to allow it to use these other types of weapons.

Reply:
Sounds good, I will remove the pylons, then. 

Much as I like diversity and half the point of pylons is to allow people to keep more primary guns while still having the option to add in secondaries... The other half is to allow ships to have more personality and defined characteristics, such as being able to say "This ship was built for guns, and nothing but guns" (or whatever the situation may be).